buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt

```diff
@@ -39,5 +39,13 @@ object IntellijVersions {
     IntellijVersionHelper.getMajorPart(latestStable)
   }
 
-  val buildTarget: String = eapOfLatestSupportedMajor ?: latestStable
+  private val buildTarget: String = eapOfLatestSupportedMajor ?: latestStable
```

Comment:
Actually... `buildTarget` isn't used for anything now, other than for `getBuildTarget` right?

Maybe sth like
```suggestion
  val buildTarget: String = overrideBuildTarget ?: eapOfLatestSupportedMajor ?: latestStable
```

Reply:
(and remove `getBuildTarget`)